src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="http://localhost:8080/"
+      method="post"
+      class="formBar"
+      data-qa="big"
+    >
+      <label>
+        <img
+          class="icon icon--big"
+          src="images/Search.svg"
+          alt="Logo image"
+        >
+        <input
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
        >
        
        <input
```